source/clear-linux/tutorials/telemetry-e2e.rst

```diff
@@ -0,0 +1,294 @@
+.. _telemetry-e2e:
+
+
+|CL-ATTR| includes a telemetry and analytics solution (also known as telemetrics) as part of the OS, which records events of interest and reports them back to the development team using the telemetrics client daemons.
+End users can enable or disable the telemetry client component of |CL| and also redirect where records are sent if they wish to collect records for themselves by using their own telemetry backend server.
+
+This tutorial walks you through creating a telemetry enabled app on your local |CL| machine. The tutorial will show you how to send your own custom reports to any telemetry backend server.
+
+.. note::
+
+   The telemetrics functionality adheres to `Intel privacy policies`_ regarding the collection and use of :abbr:`PII (Personally Identifiable Information)` and is Open Source. Specifically, no intentionally identifiable information about the user or system owner is collected.
+
+Prerequisites
+=============
+
+For this tutorial, start with a clean installation of |CL| on a new system
+using the :ref:`bare-metal-install` getting started guide and:
+
+#. Choose the automatic install.
+#. Join the :guilabel:`Stability Enhancement Program` during the installation process to enable the telemetrics client components.
+
+You can also set up a telemetry backend server; you can use the same |CL| system we're working with for this tutorial, or setup on a separate system.  Follow the :ref:`telemetry-backend` guide to setup the server and redirect your telemetry records to it. Refer to the :ref:`telemetry-config` guide to configure the telemetry client and specify where to send the records.
```

Comment:
Since this is outward facing the assumption should be that someone doing this won't be able to get to the internal telemetry server.  We should probably suggest they either set up their own server on their system so they can use localhost or go into the conf file and turn off delivery to the server and turn on record retention to enable journaling:
record_server_delivery_enabled=false
record_retention_enabled=true

Thoughts?

Reply:
I can clean this up -- make it clear that the user won't be able to access their records on the default server.  The next paragraph points people back to the telemetry config guide page to show how to set up for local retention.

Reply:
FYI: I plan to work on this branch for syntax cleanup. So, please don't worry about making those edits.